lib/dbservice_web/controllers/session_schedule_controller.ex

```diff
@@ -0,0 +1,131 @@
+defmodule DbserviceWeb.SessionScheduleController do
+  use DbserviceWeb, :controller
+
+  import Ecto.Query
+  alias Dbservice.Repo
+  alias Dbservice.SessionSchedules
+  alias Dbservice.Sessions.SessionSchedule
+
+  action_fallback(DbserviceWeb.FallbackController)
+
+  use PhoenixSwagger
+
+  alias DbserviceWeb.SwaggerSchema.SessionSchedule, as: SwaggerSchemaSessionSchedule
+
+  def swagger_definitions do
+    Map.merge(
+      SwaggerSchemaSessionSchedule.session_schedule(),
+      SwaggerSchemaSessionSchedule.session_schedules()
+    )
+  end
+
+  swagger_path :index do
+    get("/api/session-schedule")
+
+    parameters do
+      params(:query, :string, "The id of a session",
+        required: false,
+        name: "id"
+      )
+    end
+
+    response(200, "OK", Schema.ref(:SessionSchedules))
+  end
+
+  def index(conn, params) do
```

Comment:
If we have days of week and start and end time, can we store them so that it doesn't require any reformatting on client end? Maybe not, actually -- but something to think about

Reply:
I'm sorry I didn't get the point. Can you please explain?